extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -192,11 +179,23 @@ export class AmpAdApiHandler {
    * @private
    */
   updateSize_(height, width, source, origin) {
-    this.baseInstance_.attemptChangeSize(height, width).then(() => {
-      this.sendEmbedSizeResponse_(
-        true /* success */, width, height, source, origin);
-    }, () => this.sendEmbedSizeResponse_(
-        false /* success */, width, height, source, origin));
+    let newHeight, newWidth;
+    if (height !== undefined) {
+      newHeight = Math.max(this.element_./*OK*/offsetHeight +
+          height - this.iframe_./*OK*/offsetHeight, height);
+    }
+    if (width !== undefined) {
+      newWidth = Math.max(this.element_./*OK*/offsetWidth +
+          width - this.iframe_./*OK*/offsetWidth, width);
+    }
+
+    if (newHeight !== undefined || newWidth !== undefined) {
+      this.baseInstance_.attemptChangeSize(newHeight, newWidth).then(() => {
+        this.sendEmbedSizeResponse_(
+          true /* success */, newWidth, newHeight, source, origin);
+      }, () => this.sendEmbedSizeResponse_(
+          false /* success */, newWidth, newHeight, source, origin));
```

Comment:
`/*success*/` is the description of the value. Here it means `success` is `false`